ipatests/test_integration/test_user_permissions.py

```diff
@@ -93,3 +93,36 @@ def test_stageuser_show_as_alternate_admin(self):
         # the field Kerberos Keys available must contain True
         result = self.master.run_command(['ipa', 'stageuser-show', stageuser])
         assert 'Kerberos keys available: True' in result.stdout_text
+
+    def test_delete_group_by_user_administrator(self):
+        """
+        Test that a user with sufficient privileges can delete group
+        This is a Automation for issue 6884
+        """
+        # Create a new testadmin user with a password
+        testadmin = 'tuser'
+        password = 'Secret123'
+        testgroup = 'gtest'
+        testadmin_password_confirmation = "%s\n%s\n" % (password,
+                                                        password)
+        self.master.run_command(['ipa', 'user-add', testadmin,
+                                 '--first', 't',
+                                 '--last', 'user',
+                                 '--password'],
+                                stdin_text=testadmin_password_confirmation)
+
+        # Add testadmin user to role "User Administrator"
+        self.master.run_command(['ipa', 'role-add-member',
+                                 '--users', testadmin,
+                                 '"User Administrator"'])
+
+        # kinit as testuser
+        self.master.run_command(['kinit', testadmin],
+                                stdin_text=testadmin_password_confirmation)
+
+        # Create a test group
+        self.master.run_command(['ipa', 'group-add', testgroup])
+
+        # Call ipa-group-del to check if user can delete group
+        result = self.master.run_command(['ipa', 'group-del', testgroup])
```

Comment:
Just a nitpick, but I would also add a check for result.returncode == 0

Reply:
@flo-renaud that should not be needed as there is "raiseonerr=True" by default.

https://github.com/encukou/pytest-multihost/blob/862f24d536ffde32cc636151e833c38d909584b2/pytest_multihost/host.py#L202

But as you told that is just a nitpick. @ndehadrai  please leave it as it is...

Reply:
@Rezney of course, you are obviously right!